### DISCUSSION TITLE
4 Hz Packet Rate Mode

### DISCUSSION THREAD

**Discussion by dev0stranger** [ExpressLRS/ExpressLRS#2974] [Ideas]
Hi everyone, I am building a 3D printed RC catamaran as a student project. Since there is a lot of signal absorption at water level, I started looking for a way to increase the signal penetration.

I found that TBS Crossfire uses 4 Hz control transmission, it activates when the lq gets too bad, and from what I understand, it helps in emergency situations.

The documentation for the LoRa SX127X transceiver says that the Spreading Factor (SF11) improves sensitivity, compared to the current minimum SF9.

My idea is to add a new Packet Rate 4 Hz to ExpressLRS, using SF11 for maximum reliability.

At the moment, I am already working on this solution, but so far I have not managed to make the receiver and transmitter work longer than 1 second. Probably, the problem is in the incorrectly calculated Time On Air TOA.

I am currently using the following settings for 4 hz mode:
{4, RADIO_TYPE_SX127x_LORA, RATE_LORA_4HZ,       SX127x_BW_XXX_KHZ, SX127x_SF_11, SX127x_CR_4_7,  10, TLM_RATIO_1_2,  2,  XXXXXX, OTA4_PACKET_SIZE, 1},

  {4, -XXX,  XXXXXX,  XXXX,   XXXX,  XXX,  XXXX,  SNR_SCALE( -6), SNR_SCALE(0.1)},

**Comment by dev0stranger** [comment]
![photo_2024-05-17_21-17-47](https://github.com/user-attachments/assets/c75eee8d-3618-428a-ae7e-558a79f021ae)
![photo_2024-05-18_22-48-16](https://github.com/user-attachments/assets/941fb2fd-d8b7-44ca-8ff1-7c355fba4f68)
The previous air-powered version was able to travel 10km on a single charge, the signal range reached up to 800 meters with my Radiomaster Pocket and RP3 receiver

**Reply by rodiaua** [reply]
Where you able to implement the emergency mode with extra low packet rate?

**Reply by dev0stranger** [reply]
> Where you able to implement the emergency mode with extra low packet rate?

I think you were purposefully looking for this information, it is related to the political situation between our countries. I can't give you more information.

**Comment by dev0stranger** [comment]
![photo_2024-09-21_21-12-14](https://github.com/user-attachments/assets/5cffd771-f894-4065-b330-c1bb40137364)
![photo_2024-09-23_02-45-10](https://github.com/user-attachments/assets/4735a5f6-a3ee-4be7-b961-1f508d87f0e6)
I am currently working on water traction for this catamaran to increase efficiency and range

**Comment by MUSTARDTIGERFPV** [comment]
At this time we're not interested in adding such a mode. Range at 50Hz is already sufficient to achieve many hundreds of kilometers.
